Library/Basic Set/Basic Set Advantages.adq

```diff
@@ -1564,7 +1564,7 @@
 					"notes": "@One species: House Cats, Macaws, etc.@"
 				}
 			],
-			"reference": "B40",
+			"reference": "B40,48",
```

Comment:
I don't believe this notation is supported. You can use "B40,B48", but even then, the result will be less than useful, since the current options are to open the first one or to open them all -- but opening them all will result in the last one only in this case, since they refer to the same book.